scripts/update_comments.py

```diff
@@ -0,0 +1,50 @@
+"""
+Update User.comments_viewed_timestamp field & comments model.
+Accompanies https://github.com/CenterForOpenScience/osf.io/pull/1762
+"""
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.auth.core import User
+from framework.transactions.context import TokuTransaction
+
+from website.models import Comment
+from website.app import init_app
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
```

Comment:
There isn't any logging in the script. It would be useful to log--at the very least--the IDs of users who were migrated.